tests/snapshots/generate_snapshots.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+import itertools
+import subprocess
+import pathlib
+
+def generate_snapshots():
+    single_styles = ["changes", "grid", "header", "numbers"]
+    collective_styles = ["full", "plain"]
+
+    for num in range(len(single_styles)):
+        for grouped in itertools.combinations(single_styles, num + 1):
+            generate_snapshot("--style={}".format(",".join(grouped)))
+
+    for style in collective_styles:
+        generate_snapshot("--style={}".format(style))
+
+    generate_snapshot("--color=always")
+    generate_snapshot("--list-languages")
+    generate_snapshot("--list-themes")
+
+def generate_snapshot(option):
+    command = "../../target/debug/bat {} sample.rs > output/{}.snapshot.txt".format(
+        option,
+        option.replace("--", "").replace(" ", "+")
+    )
+    print("generating snapshot for {}".format(option))
+    subprocess.call(command, shell=True)
+
+def prepare_output_dir():
+    output = pathlib.Path("output")
+    if output.exists():
```

Comment:
There is also [`shutil.rmtree`](https://docs.python.org/3/library/shutil.html#shutil.rmtree), which could be used with `ignore_errors=True`.